plugins/dynamix/scripts/monitor

```diff
@@ -324,30 +324,32 @@ if ($warn!="rw") {
 }
 
 // check docker image disk utilization
-$item = 'system';
-$name = 'docker';
-$last = isset($saved[$item][$name]) ? $saved[$item][$name] : '';
-$warn = str_replace("%", "", exec("df /var/lib/docker|grep -Pom1 '\d+%'"));
-if (file_exists("/boot/config/docker.cfg")) {
-  $cfg  = parse_ini_file("/boot/config/docker.cfg");
-  $info = "Docker utilization of image file {$cfg['DOCKER_IMAGE_FILE']}";
-} else
-  $info = "Docker image file not specified";
+$warn = exec("df /var/lib/docker|awk '/^\//{print $5*1}'");
```

Comment:
I added this same sort of check in 6.2 but forgot to backport it to 6.1.x.  Mine was:
```
system('mountpoint -q /var/lib/docker', $retval);
if ($retval===0) {
```